tests/robot-tests/tests/libs/tables-common.robot

```diff
@@ -23,9 +23,10 @@ user checks table row heading contains
     user waits until element contains    xpath://table/tbody/tr[${row}]/th[${column}]    ${expected}
 
 user checks table cell contains
-    [Arguments]    ${row}    ${column}    ${expected}    ${parent}=css:table
+    [Arguments]    ${row}    ${column}    ${expected}    ${parent}=css:table    ${wait}=%{WAIT_SMALL}
```

Comment:
Does this need a wait? Typically, we could use some other keyword to check that the table (or something in it) has rendered before checking the table cells.

Most 'check' keywords don't include waits for this reason, so this is fairly inconsistent.

Reply:
I suppose I'm exposing the wait param of the underlying `user waits until parent contains element`, am I to rename this to `user waits until table cell contains`?

Reply:
Was added due to table updating asynchronously to the form submit, but have abstracted the waits into separate keywords